_functions.sh

```diff
@@ -1542,10 +1553,15 @@ do_configure_apache() {
   unset DOMAIN
   echo_info "Done."
 
+  echo_info "Configure and update Matrix client well-known endpoint..."
+  mkdir -p ${MATRIX_CONF_DIR}
+  # Generate configuration for Matrix client at .well-known/matrix/client
+  evaluate_file_content ${ETC_DIR}/matrix/client.template ${MATRIX_CONF_DIR}/client
```

Comment:
${MATRIX_CONF_DIR}/client not enough. The location should have a reference to the instance. Otherwise, You will have conflicts